Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 20
Rules claiming more threads will be scaled down.
Conda environments: ignored
Singularity containers: ignored
Job counts:
	count	jobs
	1	all
	3	gzip
	3	snpEff
	1	snpEffDbDownload
	8

[Sat Sep 12 21:58:28 2020]
rule snpEffDbDownload:
    output: resources/snpeff/
    log: logs/snpEff/snpeff_download.log
    jobid: 61

/home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmplnekhjru.wrapper.py
[Sat Sep 12 21:58:47 2020]
Error in rule snpEffDbDownload:
    jobid: 61
    output: resources/snpeff/
    log: logs/snpEff/snpeff_download.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 152 of /home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk:
Command 'set -euo pipefail;  /home/snagi/miniconda3/bin/python /home/snagi/lstm_data/rna-seq-ir/.snakemake/scripts/tmplnekhjru.wrapper.py' returned non-zero exit status 1.
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2189, in run_wrapper
  File "/home/snagi/lstm_data/rna-seq-ir/workflow/rules/variants.smk", line 152, in __rule_snpEffDbDownload
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/home/snagi/miniconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/home/snagi/miniconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Removing output files of failed job snpEffDbDownload since they might be corrupted:
resources/snpeff/
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/snagi/lstm_data/rna-seq-ir/.snakemake/log/2020-09-12T215828.186049.snakemake.log
